### ISSUE TITLE
saveMessages after query.messages() is executed

### ISSUE THREAD

**Issue by samuelstroschein** [inlang/inlang#1255]
## Context

Apps shouldn't need to deal with saving messages. Thus, the inlang instance auto saves message changes that are conducted by `inlang.query.messages()`. 

## Tasks

- [ ] implement reactive `saveMessages`

**Comment by ivanhofer** [comment]
Ah ok, let me know if you need some help.

**Comment by janfjohannes** [comment]
@samuelstroschein  yes, done. i just updated the status of the ticket but let me know if you do usually do this instead or there is another final review for tickets
